twinn_ml_interface/objectmodels/hierarchy.py

```diff
@@ -19,6 +19,7 @@ class DataLevel(str, Enum):
     AVAILABILITY = "availability"
     PREDICTION = "prediction"
     ANOMALY = "anomaly"
+    LABEL = "label"
```

Comment:
What do we use this for?

Reply:
I'm also not completely sure about this. But in some functions I use it to denote that some features are of the type "LABEL". Also maybe when we want to send labels to AquasuiteAPI it could be useful to distinguish.